src/io/flutter/analytics/Analytics.java

```diff
@@ -105,6 +105,7 @@ public void sendException(Throwable throwable, boolean isFatal) {
     }
 
     String description = stringWriter.toString().trim();
+    description = description.replaceAll("com.intellij.openapi.", "c.i.o.");
```

Comment:
Did you compare the net change of
com.intellij => c.i
vs
com.intellij.openapi => c.i.o
? 
I see a lot of com.intellij and wonder if a simpler transform might give greater savings.

Reply:
The specific stack trace I saw had most things prefixed by `com.intellij.openapi.`, but I'm happy to check for the longer, than the shorter.